### PR TITLE
Add and link glossary terms UID and intid

### LINKED ISSUE #1460: Add and link glossary terms UID and intid
### Plone Documentation versions affected

- [ ] 5.x
- [X] 6.x

### Description

The terms UID and intid are used at several places. In my opinion it would be good to add them as glossary terms and  link to the glossary everywhere used.

### PR DESCRIPTION
**Pull request by lord2anil**
glossary added for UID and intid.
 closes #1460

### CODE REVIEW

File: docs/glossary.md
Comment (stevepiercy) [inline docs/glossary.md:647]: We can stack terms.

```suggestion
unique identifier
UID
```

File: docs/glossary.md
Comment (stevepiercy) [inline docs/glossary.md]: In this context, we are assigning a unique identifier to an image object, which is independent of a user identifier.

```suggestion
   UID is an acronym meaning "unique identifier".
   A UID is an identifier that is guaranteed to be unique among all identifiers used for those objects and for a specific purpose.
```

File: docs/glossary.md
Comment (stevepiercy) [inline docs/glossary.md:652]: Where's the linked term?

Comment (stevepiercy) [inline docs/glossary.md:652]:
```suggestion
integer identifier
intid
```

File: docs/glossary.md
Comment (stevepiercy) [inline docs/glossary.md]:
```suggestion
    In Plone, an integer identifier, or intid, is used to uniquely identify content objects within a Plone site.
    Each content item in a Plone site is given a unique intid, which the system uses internally to monitor and manage content.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Please see review comments. Thank you!

File: docs/glossary.md
Comment (jensens) [reply docs/glossary.md]: "monitor and manage content"? 
I would say: reference content, keep track of link-integrity, link translations and related.

**Review by stevepiercy — APPROVED** [review]
I approve, assuming that you will incorporate @jensens's suggestions as well. Thank you!

**Comment by stevepiercy** [comment]
@lord2anil would you please update this PR? Thank you!

File: docs/glossary.md
Comment (stevepiercy) [inline docs/glossary.md:654]: Per @jensens's comment.

```suggestion
    Each content item in a Plone site is given a unique intid, which the system uses internally to reference content, keep track of link-integrity, link translations, and other related purposes.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
I think this is what @jensens intended, can you verify?

Also please resolve conflicts in the file `docs/glossary.md`. This PR cannot be merged until that is done. Thank you!

**Comment by lord2anil** [comment]
i don't have write access to branch .The created a new PR form  Fork repo please have a look. https://github.com/plone/documentation/pull/1487/#issue-1659752403

**Comment by stevepiercy** [comment]
Ah, sorry, that's right, we had to restrict access to new contributors. There was an announcement in the Community Forum, and we documented details in [First-time contributors](https://6.docs.plone.org/contributing/first-time.html). Closing this PR in favor of #1487.

**Pull request closed without merging**
